docs/source/tracking.rst

```diff
@@ -250,17 +250,19 @@ You run an MLflow tracking server using ``mlflow server``.  An example configura
 Storage
 -------
 
-An MLflow tracking server has two properties related to how data is stored: file store and artifact store.
+An MLflow tracking server has two properties related to how data is stored: a backend store and
+an artifact store.
 
 The *backend store* (exposed as ``--backend-store-uri``) is where the *server* stores run and
 experiment metadata. For backward compatibility, ``--file-store`` option is an alias to this
-option. This can be a local path **file store** specified as ``./path_to_store`` or
-``file://path_to_store``, or a SQL connection string for a *Database backed store*. For the
-latter, argument is expected to be a SQL connection string specified as
+option. This can be a local path *file store* specified as ``./path_to_store`` or
+``file://path_to_store``, or a SQL connection string for a *database backed store*. For the
```

Comment:
Nit: I think there should be a hyphen here: `database-backed store`. @stbof ?

Reply:
Yes.